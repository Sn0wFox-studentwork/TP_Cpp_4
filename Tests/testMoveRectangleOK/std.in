R r1 2 2 0 0
MOVE r1 1 1
LIST
EXIT