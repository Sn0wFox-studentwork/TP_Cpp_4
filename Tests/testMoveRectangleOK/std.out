OK
OK
R r1 3 3 1 1